### ISSUE TITLE
Provide table with initial data

### ISSUE THREAD

**Issue by niktekusho** [typeorm/typeorm#1763]
**Issue type:**

[x] question
[ ] bug report
[x] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[x] `postgres`
[ ] `sqlite`
[ ] `sqljs`
[ ] `websql`

**TypeORM version:**

[x] `latest`
[ ] `@next`
[ ] `0.x.x` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**
Hi guys, awesome project 👍.
I have an ```@Entity``` in which I create a "uuid" auto generated column and other simple columns and I want to fill in some default data right after the actual table creation. The following is a simplified sample of the entity I am dealing with.
```ts
@Entity()
export class TestEntity {

    @PrimaryGeneratedColumn()
    @Generated("uuid")
    id: number;

    @Column()
    data: string;
}
```

I already tried using the appropriate repository's ```preload()``` method passing in something like
```{ data: 'whatever' }```, but it failed on me saying that it misses the Primary column. Exact error:

```Given object does not have a primary column, cannot transform it to database entity.```

Since the id is generated, I would like to add the initial data without specifing it. It is only used internally and it does not need to be known outside of the database world.

Would it be possible to add a method that does those SQL ```INSERT```s right after the table creation?
I think that a **commonish** use case for this feature is to have a json file, containing the default data I want to insert (could also be an array representing rows), which gets parsed and passed into this method. When this method finishes they should also be committed to the database.

**Comment by vados-p7e** [comment]
Just curious, have you tried doing this in `@next`?

**Comment by niktekusho** [comment]
@vados-p7e  nope... I'll try it in these days 😄

**Comment by vados-p7e** [comment]
@niktekusho just to be clear, I'm not sure it would fix it (the code looks to not take it into account, tests seem so require `id` to be specified), but thought it might be worth a try :)

**Comment by niktekusho** [comment]
Just gave it a try. As @vados-p7e anticipated, ```@next``` could not fix it.
I took a look at ```PlainObjectToDatabaseEntityTransformer``` and I found the line which checks the object's validity:
https://github.com/typeorm/typeorm/blob/97235df29dc50d9205f264c4e309b3c691d16596/src/query-builder/transformer/PlainObjectToDatabaseEntityTransformer.ts#L92

Later I might have a look at ```EntityMetadata``` in order to understand if what I want could be done.
